### PR TITLE
Update lottie-ios.podspec

### PR DESCRIPTION
**Pull request by romainbiard**
Adding tvos support

Thanks

### CODE REVIEW

File: lottie-ios.podspec [hunk 1]
```diff
@@ -24,6 +24,7 @@ For the first time, designers can create and ship beautiful animations without a
 
   s.ios.deployment_target = '8.0'
   s.osx.deployment_target = '10.10'
+  s.tvos.deployment_target = '9.0'
 
   s.source_files = 'lottie-ios/Classes/**/*'
```
Comment (montee33) [inline lottie-ios.podspec]: N

Comment (montee33) [inline lottie-ios.podspec]: ios

Comment (romainbiard) [inline lottie-ios.podspec]: Sorry I didn't catch you, you mean ?

Comment (buba447) [inline lottie-ios.podspec]: Does this work on tvOS without crashing? I haven't been up to date on the supported features of tvOS @romainbiard

Comment (romainbiard) [inline lottie-ios.podspec]: It works without crashing even if I didn't performed any exhaustive tests to be 100% sure of that. :-)

**Pull request merged**